### PR TITLE
Make Logging Unobserved Task Exceptions Optional

### PR DESCRIPTION
**Pull request by spencerlandis**
UnobservedTaskException can be handled in a way that prevents the app from crashing. 

If this is the case, don't log the exception as a crash! Leave it enabled by default, but allow it to be optional.

### CODE REVIEW

**Comment by TroubleMakerBen** [comment]
hey @spencerlandis,

thx for the contribution =) That's awesome. We're going to look into this in the new year and be in touch.

Happy holidays and thx for using HockeyApp =)

Benjamin

File: source/HockeySDK.AndroidBindings/Additions/CrashManager.cs [hunk 1]
```diff
@@ -9,14 +9,21 @@ public partial class CrashManager
...
+		private static bool terminateOnUnobservedTaskException = false;
+        	private static bool logUnobservedTaskException = true;
 
 		public static bool TerminateOnUnobservedTaskException
 		{
 			get { return terminateOnUnobservedTaskException; }
 			set { terminateOnUnobservedTaskException = value; }
 		}
 
+		public static bool LogUnobservedTaskException
```
[the remark this answers is not in the document]
Comment (spencerlandis) [reply source/HockeySDK.AndroidBindings/Additions/CrashManager.cs]: Yep, updated.

**Comment by YvesDelcoigne** [comment]
Can this PR be merged? It would help make our crash reporting actually useful.

**Comment by TroubleMakerBen** [comment]
Hey @YvesDelcoigne,
`It would help make our crash reporting actually useful.` are you saying that you don't want the HockeySDK to not support the UnobservedTaskExceptions as crashes at all?

**Comment by YvesDelcoigne** [comment]
Hi @TroubleMakerBen , no, I'm fine with making it optional like this PR is implemented, but I would like to see it merged and a new nuget made available, so we can use it.

Right now, our project is dependent on a library that is throwing task exceptions by design, so our crashes' current count is almost 1000k, while the actual crashes are impossible to find in this heap.

**Comment by guperrot** [comment]
#131 replaces this change.

**Pull request closed without merging**